curriculum/challenges/english/14-responsive-web-design-22/learn-basic-css-by-building-a-cafe-menu/5f3477cbcb6ba47918c1da92.md

```diff
@@ -20,21 +20,21 @@ Add the following within the `head` element:
 Your code should have two `meta` elements.
 
 ```js
-assert(code.match(/<meta.*?\/?>/g).length === 2);
+assert.lengthOf(code.match(/<meta.*?\/?>/g),2);
 ```
 
 Your `meta` element should have a `name` attribute with a value of `viewport`.
 
 ```js
-const meta = $('meta');
-assert(meta[0].outerHTML.match(/name=('|")viewport\1/) || meta[1].outerHTML.match(/name=('|")viewport\1/));
+const meta = document.querySelectorAll('meta');
+assert.exists(meta[0]?.outerHTML?.match(/name=('|")viewport\1/) || meta[1]?.outerHTML?.match(/name=('|")viewport\1/));
```

Comment:
Sorry for these two (one above and below), I actually thought `isTrue` would accept truthy value. Because of that I think it could be left with _bare_ `assert`.

I've also looked at other _asserts_, if there might be some better suited for it than `assert.exists`. `assert.isNotNull` is literally asserting _not null_, or `assert.isOk` - accepts truthy values (which is kind of what does _bare_ `assert`.